src/LykkePublicAPI/Controllers/MarketController.cs

```diff
@@ -160,25 +160,15 @@ public class MarketController : Controller
             var to = DateTime.UtcNow; // exclusive
             var from = to - TimeSpan.FromHours(25); // inclusive
 
-            var candles = await Policy
-                .Handle<Exception>()
-                .WaitAndRetryAsync(new[]
-                    {
-                        TimeSpan.Zero,
-                    },
-                    async (ex, ts, c) =>
-                    {
-                        assetPairs = await GetAvailableAssetPairsAsync(market);
-                    })
-                .ExecuteAsync(() =>
+            if (!assetPairs.Contains(assetPair))
+            {
+                return new CandlesHistoryResponseModel
```

Comment:
Let's log such event at least on info level.

Reply:
It's absolutely normal situation since MT and spot market have different assets set. Don't think it should be logged at all.